core/include/detray/geometry/surface.hpp

```diff
@@ -116,35 +121,11 @@ class surface {
 
     /// Access to the mask
     DETRAY_HOST_DEVICE
-    auto mask() -> const mask_link & { return _mask; }
-
-    /// @return the mask link
-    DETRAY_HOST_DEVICE
-    auto mask() const -> const mask_link & { return _mask; }
-
-    /// Access to the mask id
-    DETRAY_HOST_DEVICE
-    auto mask_type() -> typename mask_link::id_type {
-        return detail::get<0>(_mask);
-    }
-
-    /// @return the mask link
-    DETRAY_HOST_DEVICE
-    auto mask_type() const -> typename mask_link::id_type {
-        return detail::get<0>(_mask);
-    }
-
-    /// Access to the mask
-    DETRAY_HOST_DEVICE
-    auto mask_range() -> typename mask_link::index_type & {
-        return detail::get<1>(_mask);
-    }
+    constexpr auto mask() -> const mask_link & { return _mask; }
```

Comment:
Just wondering: why don't you write a function like `constexpr const mask_link& mask()`?

Reply:
A matter of taste, I guess. When I have a lot of functions with implementations in a header, I like to have a more uniform alignment, like
```
auto func1 ....
auto func2 ....
auto func3 ....
```
Like this, I can read the function names faster when I search something specific, but that might be a personal thing :D